custom-configurations-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/requests/CloneCustomConfigsV4Request.java

```diff
@@ -0,0 +1,43 @@
+package com.sequenceiq.cloudbreak.api.endpoint.v4.requests;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.sequenceiq.cloudbreak.doc.ModelDescriptions;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CloneCustomConfigsV4Request {
```

Comment:
should we rename this as well ?

Reply:
Acknowledged, will rename the API request and response models.